tests/test_py35/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
```

Comment:
i explained that in the file:

> This file allows files in here to be called the same as other test files.

if there is e.g. `/tests/test_streams` and `/tests/test_py35/test_streams`, then `/tests/test_py35/__init__.py` *has* to exist due to the module cache. try it if you don’t believe me :smile: